cdk-erigon-sequencer-params.yal

```diff
@@ -0,0 +1,181 @@
+---
+# Deployment parameters.
+# The deployment process is divided into various stages.
+
+# Deploy local L1.
+deploy_l1: true
+
+# Deploy zkevm contracts on L1 (and also fund accounts).
+deploy_zkevm_contracts_on_l1: true
+
+# Deploy zkevm node and cdk peripheral databases.
+deploy_databases: true
+
+# Deploy cdk central/trusted environment.
+deploy_cdk_central_environment: true
+
+# Deploy cdk/bridge infrastructure.
+deploy_cdk_bridge_infra: true
+
+# Deploy permissionless node.
+deploy_zkevm_permissionless_node: false
+
+# Deploy cdk-erigon node.
+deploy_cdk_erigon_node: true
+
+# Deploy observability stack.
+deploy_observability: true
+
+# Deploy Blockscout stack.
+deploy_l2_blockscout: false
+
+# Deploy eth load balancer.
+deploy_blutgang: false
+
+# Apply workload to the different components of the CDK stack.
+apply_workload: false
+
+args:
+  # Suffix appended to service names.
+  # Note: It should be a string.
+  deployment_suffix: "-001"
+
+  # Sequence type. Choose from ["erigon", "zkevm-node"]
+  sequencer_type: "erigon"
+  # The type of data availability to use.
+  # Options:
+  # - 'rollup': Transaction data is stored on-chain on L1.
+  # - 'cdk-validium': Transaction data is stored off-chain using the CDK DA layer and a DAC.
+  # In the future, we would like to support external DA protocols such as Avail, Celestia and Near.
+  data_availability_mode: cdk-validium
+
+  # Docker images and repositories used to spin up services.
+  zkevm_prover_image: hermeznetwork/zkevm-prover:v6.0.3-RC9
+  # zkevm_prover_image: hermeznetwork/zkevm-prover:v4.0.19
+  zkevm_node_image: hermeznetwork/zkevm-node:v0.6.7
+  cdk_node_image: 0xpolygon/cdk-validium-node:0.6.7-cdk
+
+  zkevm_da_image: 0xpolygon/cdk-data-availability:0.0.7
+  # zkevm_da_image: 0xpolygon/cdk-data-availability:0.0.6
+  zkevm_contracts_image: leovct/zkevm-contracts # the tag is automatically replaced by the value of /zkevm_rollup_fork_id/
+
+  # zkevm_agglayer_image: 0xpolygon/agglayer:0.1.3
+  zkevm_agglayer_image: ghcr.io/agglayer/agglayer-rs:0.1.0
+  zkevm_bridge_service_image: hermeznetwork/zkevm-bridge-service:v0.4.2
+  cdk_erigon_node_image: hermeznetwork/cdk-erigon:latest
+  panoptichain_image: minhdvu/panoptichain
+  zkevm_bridge_ui_image: leovct/zkevm-bridge-ui:multi-network
+  zkevm_bridge_proxy_image: haproxy:2.9.7
+  workload_image: leovct/workload:0.0.1
```

Comment:
nit: ideally we can make this references to official Polygon repos in time @leovct